cmd/terramate/cli/cloud.go

```diff
@@ -1053,6 +1205,43 @@ func (c *cli) newReviewRequest(
 	return rr
 }
 
+func (c *cli) newGitlabReviewRequest(mr gitlab.MR) *cloud.ReviewRequest {
+	if c.cloud.run.rrEvent.pushedAt == nil {
+		panic(errors.E(errors.ErrInternal, "CI pushed_at is nil"))
+	}
+	mrUpdatedAt, err := time.Parse(time.RFC3339, mr.UpdatedAt)
+	if err != nil {
+		printer.Stderr.WarnWithDetails("failed to parse MR.updated_at field", err)
+
+		t := *c.cloud.run.rrEvent.pushedAt
+		mrUpdatedAt = t
+	}
+	rr := &cloud.ReviewRequest{
+		Platform:    "gitlab",
+		Repository:  c.prj.prettyRepo(),
+		URL:         mr.WebURL,
+		Number:      mr.IID,
+		Title:       mr.Title,
+		Description: mr.Description,
+		CommitSHA:   c.prj.headCommit(),
+		Draft:       mr.Draft,
+		PushedAt:    c.cloud.run.rrEvent.pushedAt,
+		UpdatedAt:   &mrUpdatedAt,
+		Status:      mr.State,
+		Author: cloud.Author{
+			Login:     mr.Author.Username,
+			AvatarURL: mr.Author.AvatarURL,
+		},
```

Comment:
Missing the branch, base branch and created_at from here, right?

Reply:
this was resolved in the gitlab repo, I forgot to push here, sorry.